ui/components/app/flask/snap-settings-card/snap-settings-card.js

```diff
@@ -0,0 +1,287 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import Card from '../../../ui/card';
+import Box from '../../../ui/box';
+import SiteIcon from '../../../ui/site-icon';
+import Typography from '../../../ui/typography/typography';
+import ToggleButton from '../../../ui/toggle-button';
+import Chip from '../../../ui/chip';
+import ColorIndicator from '../../../ui/color-indicator';
+import Button from '../../../ui/button';
+
+import {
+  COLORS,
+  TYPOGRAPHY,
+  FONT_WEIGHT,
+  ALIGN_ITEMS,
+  DISPLAY,
+  TEXT_ALIGN,
+} from '../../../../helpers/constants/design-system';
+
+const STATUSES = {
+  INSTALLING: 'installing',
+  RUNNING: 'running',
+  STOPPED: 'stopped',
+  CRASHED: 'crashed',
+};
+
+const STATUS_COLORS = {
+  [STATUSES.INSTALLING]: COLORS.ALERT1,
+  [STATUSES.RUNNING]: COLORS.SUCCESS1,
+  [STATUSES.STOPPED]: COLORS.UI4,
+  [STATUSES.CRASHED]: COLORS.ERROR1,
+};
+
+const SnapSettingsCard = ({
+  name,
+  description,
+  iconUrl,
+  dateAdded,
+  version,
+  url,
+  onToggle,
+  isEnabled = false,
+  onClick,
+  status,
+  className,
+  cardProps,
+  toggleButtonProps,
+  buttonProps,
+  chipProps,
+}) => {
+  const [chipStatus, setChipStatus] = useState({
+    label: STATUSES.INSTALLING,
+    indicatorColor: STATUS_COLORS[STATUSES.INSTALLING],
```

Comment:
Looking at how `indicatorColor` is used, it always mirrors its status.  Do we need the separate key/value?

Reply:
Good catch, I overlooked this in my initial review. Thanks!